ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/zones/provision/cucushift-installer-rehearse-nutanix-ipi-zones-provision-chain.yaml

```diff
@@ -0,0 +1,13 @@
+chain:
+  as: cucushift-installer-rehearse-nutanix-ipi-zones-provision
+  steps:
+  - chain: ipi-conf-nutanix
+  - ref: ipi-conf-nutanix-zones
+  - chain: ipi-install
+  - ref: ipi-install-nutanix-csi
+  - ref: ipi-install-post-monitoringpvc
```

Comment:
If we are installing monitoringpvc as post install actions, then we are not supposed to configure `PERSISTENT_MONITORING: "false"`.

Reply:
there's some issue with nutanix cis, see here https://redhat-internal.slack.com/archives/C0211848DBN/p1706846976089119
So I asked monitoring team to use this as a workaround

Reply:
Somehow I can not access the slack link, can you confirm the link is right?
And I check another job - `nutanix-ipi-image-registry-s3-f28` which is referencing `ipi-install-nutanix-csi` and `ipi-install-post-monitoringpvc`, for its [log](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-nutanix-ipi-image-registry-s3-f28/1753627273853931520), seem it works well.

Reply:
I invited you in slack, zones job run on a new env where there have multi PEs

Reply:
Thanks, if the nutanix-csi is not ready for failure-domain cases, I think we can directly remove `ipi-install-nutanix-csi` and `ipi-install-post-monitoringpvc`.

Reply:
I think ipi-install-nutanix-csi and ipi-install-post-monitoringpvc can be moved too, if use configure-registry-storage-nutanix step, because the step, the registry storage is used Emptydir

Reply:
About the csi issue, seems it's related talk here: https://github.com/kubernetes-sigs/vsphere-csi-driver/issues/2488#issuecomment-1688093660, the issue should be resolved for more time, so sometimes we meet it works, but some times doesn't

Reply:
Ack, removed all steps in chain   - chain: ipi-install-nutanix

Reply:
@sgaoshang I think we need to keep `configure-registry-storage-nutanix` step.

Reply:
@wewang58 is my comment correct?

Reply:
@jianlinliu  Yes, the step needs

Reply:
no  need step:  ipi-install-nutanix-csi and ipi-install-post-monitoringpvc.

Reply:
from the test [job](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/47854/rehearse-47854-periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-nutanix-ipi-zones-f14/1754706840161095680), it get passed, if we do not set it, what would happen?

Reply:
@sgaoshang  Per wenwang's private reply, if not run `configure-registry-storage-nutanix`, the image-registry would be in `removed` state, that means, no image-registry available to use in the cluster.

Reply:
Added it, thanks.